tests/benchmarks/test_poisson/poisson.cc

```diff
@@ -204,7 +204,7 @@ int main (int argc, char **argv)
 
   try
     {
-      Utilities::System::MPI_InitFinalize mpi_initialization(argc, argv);
+      Utilities::MPI::MPI_InitFinalize mpi_initialization (argc, argv, testing_max_num_threads());
```

Comment:
*ugh*
I did not know that we still have <code>Utilities::System::MPI_InitFinalize</code> around..